packages/Icon/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/icon",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Did you make this change?  You don't need to, the release will bump it automatically.

Reply:
Thanks, will revert these! 🙏